### ISSUE TITLE
Issues with sqlite:  file not recognized: File format not recognized

### ISSUE THREAD

**Issue by Martengg** [vlang/v#7661]
Hey folks, 
I start to get depressed because I'm not able to execute these lines of code. I am not shure if this is a valid bug - Can you help me? Thank you in advance!

`v doctor`

OS: windows, Microsoft Windows 10 Pro v18363 64-Bit
Processor: 4 cpus, 64bit, little endian, Intel(R) Core(TM) i7-3520M CPU @ 2.90GHz
CC version: N/A

getwd: C:\Users\angel
vmodules: C:\Users\angel\.vmodules
vroot: D:\Programme\v
vexe: D:\Programme\v\v.exe
vexe mtime: 2020-12-28 21:00:24
is vroot writable: true
is vmodules writable: true
V full version: V 0.2 954580f

Git version: git version 2.27.0.windows.1
Git vroot status: 0.2-140-g954580f8
.git/config present: true
thirdparty/tcc status: thirdparty-windows-amd64 8836aa8f

**What did you do?**
I tried to compile the following code:

```
import sqlite

struct Groups {
	id int
	day int
	name string
}
db := sqlite.connect('test.db')?


count_groups := sql db { select count from Groups }
println("number of all groups: $count_groups")
```

**What did you expect to see?**

I expected to see the number of all groups. The code is fully working. I am working in a team where a friend compiled it on a Linux machine and it was fully working.

**What did you see instead?**
```
D:\Dateien\Programmieren\v\vlang_plg>v run sqlite-orm.v
D:\Programme\v\thirdparty\sqlite\sqlite3.o not found, building it in C:\Users\angel\.vmodules\cache\10\10c2f092da3805de7b39c50aee497a43.o ...

Compilation with tcc failed. Retrying with gcc ...
==================
C:\Users\angel\.vmodules\cache\10\10c2f092da3805de7b39c50aee497a43.o: file not recognized: File format not recognized 
collect2.exe: error: ld returned 1 exit status
...
==================
(Use `v -cg` to print the entire error message)

builder error:
==================
C error. This should never happen.

If you were not working with C interop, please raise an issue on GitHub:

https://github.com/vlang/v/issues/new/choose

You can also use #help on Discord: https://discord.gg/vlang```
```

**Other information**

- I installed minGW as a gcc compiler
- I installed the sqlite library as explained in here: https://modules.vlang.io/sqlite.html
- I installed sqlite3 for windows
- `sqlite3`, `gcc` and `v` are available as commands in cmd

**Comment by spaceface777** [comment]
Could you please try these two things:
 1. these will most likely not make the example compile for you, but it will at least give us more debugging info as to what's causing the issue.
     - `rmdir /s /q "%USERPROFILE%\.vmodules\cache"` (wipes V's module cache)
     - run `v -show-c-output -cg -v run sqlite-orm.v` and upload the output somewhere

 2. 
     - `rmdir /s /q "%USERPROFILE%\.vmodules\cache"`
     - `v -cc gcc run sqlite-orm.v` (forces V to use gcc)


***
some quick notes for V devs:
- V seems to be passing the tcc `.o` to gcc after a failed compilation; this needs to be fixed
- *we should really print the reason why tcc failed* rather than asking the user to pass `-show-c-output` every time
- a `v wipe-cache` (or some other name for it, idk) command would be quite useful and easy to add imho, at least until module caching stabilizes a little

@spytheman ^   🙂

**Comment by Martengg** [comment]
Thank you for your fast reply! I deleted the cache and uploaded the full output in [here](https://pastebin.com/2NU6dFCk).

And thank you very much for your workaround! That works as a temporary solution.

**Comment by spytheman** [comment]
I just implemented `v wipe-cache` in 8724749 @spaceface777 .

**Comment by spaceface777** [comment]
Fixed in vlang/tccbin@bac32fd.
